app/build.gradle

```diff
@@ -14,6 +20,11 @@ android {
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        buildConfigField("String", "API_BASE_URL", apiProperties['API_BASE_URL'])
+        buildConfigField("String", "MARVEL_PRIVATE_KEY", apiProperties['MARVEL_PRIVATE_KEY'])
+        buildConfigField("String", "MARVEL_PUBLIC_KEY", apiProperties['MARVEL_PUBLIC_KEY'])
+        buildConfigField("String", "TS", apiProperties['TS'])
```

Comment:
Can we put this config fields in the data module?

Reply:
Sure, we can, but  you told us do not use the other layers (waiting for coders section)
Please confirm for move it.